sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/util/BinaryData.java

```diff
@@ -185,16 +185,44 @@ public static BinaryData fromBytes(byte[] data) {
         return new BinaryData(data);
     }
 
+    /**
+     * Serialize the given {@link Object} into {@link BinaryData} using json serializer which is available on classpath.
+     * The serializer on classpath must implement {@link JsonSerializer} interface. If the given Object is {@code null},
+     * an empty {@link BinaryData} will be returned.
```

Comment:
This documentation should link through an aka.ms link to the appropriate json serializer documentation (which will soon move to docs.microsoft.com, which is why in the meanwhile we will use an aka.ms link)

Reply:
Added this link https://aka.ms/azsdk/java/wiki/serialization